app/src/test/java/org/simple/clinic/appconfig/AppConfigRepositoryTest.kt

```diff
@@ -30,4 +36,83 @@ class AppConfigRepositoryTest {
         .assertValue(FetchSucceeded(countries))
         .assertNoErrors()
   }
+
+  @Test
+  fun `failed network calls to fetch the app manifest to return an appropriate failure result`() {
+    // given
+    val cause = ConnectException()
+    whenever(api.fetchManifest()).thenReturn(Single.error(cause))
```

Comment:
Are we using `thenReturn` or moving to `doReturn`?

Reply:
I managed to do something to remove the IDE errors. I'll use this for a couple more days and see. If the errors have gone, I'll revert to `thenReturn` everywhere else.